packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx

```diff
@@ -93,12 +106,31 @@ export const DeclarativeChart: React.FunctionComponent<DeclarativeChartProps> =
     ...(activeLegends.length > 0 && { selectedLegend: activeLegends[0] }),
   };
 
+  const exportAsImage = React.useCallback(
+    (opts?: IImageExportOptions) => {
+      return toImage(chartRef.current?.container, {
+        background: theme.palette.white,
```

Comment:
theme can be null if used in v9 directly

Reply:
useTheme hook always returns a theme: https://github.com/microsoft/fluentui/blob/622c4aacdb5a1ab601287a9adb1064a640388f4f/packages/react/src/utilities/ThemeProvider/useTheme.ts#L21